components/library/cairo/Makefile

```diff
@@ -20,11 +20,11 @@ DEFAULT_COMPONENT_TEST_TRANSFORMS= yes
 include ../../../make-rules/shared-macros.mk
```

Comment:
I need to correct my own error. The correct setting is
USE_DEFAULT_TEST_TRANSFORMS= yes
Please fix this, too.